examples/hpa/README.md

```diff
@@ -1,3 +1,169 @@
+# custom-metrics-stackdriver-adapter example instructions
+
+The Custom Metrics Stackdriver Adapter supports querying metrics from
+Managed Service for Prometheus starting with
+[version v0.13.1 of the adapter](https://github.com/GoogleCloudPlatform/k8s-stackdriver/releases/tag/cm-sd-adapter-v0.13.1).
+
+To set up an example HPA configuration using the Custom Metrics Stackdriver
+Adapter, do the following:
+
+1. Set up Managed Service for Prometheus in your cluster.
+2. Install Custom Metrics Stackdriver Adapter in your cluster.
+
+   ```sh
+   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/deploy/production/adapter_new_resource_model.yaml
+   ```
+
+3. Deploy an example Prometheus metrics exporter and an HPA resource:
+
+   ```sh
+   kubectl apply -f https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/custom-metrics-prometheus-sd.yaml
```

Comment:
I think we discussed to use [https://www.google.com/url?sa=D&q=https%3A%2F%2Fgithub.com%2FGoogleCloudPlatform%2Fk8s-stackdriver%2Fblob%2Fmaster%2Fcustom-metrics-stackdriver-adapter%2Fexamples%2Fdirect-to-sd%2Fcustom-metrics-sd.yaml instead?](https://github.com/GoogleCloudPlatform/k8s-stackdriver/blob/master/custom-metrics-stackdriver-adapter/examples/direct-to-sd/custom-metrics-sd.yaml) instead?

Reply:
That doesn't work as it emits the metric in k8s_container format, not GMP format

Reply:
I think we could use https://raw.githubusercontent.com/GoogleCloudPlatform/prometheus-engine/v0.5.0/examples/example-app.yaml instead?

Reply:
I am ok to use the current https://raw.githubusercontent.com/GoogleCloudPlatform/k8s-stackdriver/master/custom-metrics-stackdriver-adapter/examples/prometheus-to-sd/custom-metrics-prometheus-sd.yaml if it emits the expected metric but I think the example contains a redundant container prometheus-to-sd, which is not in use.

Reply:
looks like `example-app` doesn't emit any static-valued metrics besides `up`, and those are most useful for this scenario. Can always change it to use example-app later I suppose.